modules/backend/classes/BackendController.php

```diff
@@ -111,6 +111,14 @@ public static function extend(Closure $callback)
         self::extendableExtendCallback($callback);
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function callAction($method, $parameters)
+    {
+        return parent::callAction($method, array_values($parameters));
```

Comment:
In php8 named parameters were introduced and now it is required to match called method parameter name when setting parameters by array destructing or `call_user_func_array()` etc.